packages/discord-verify/src/middleware/express.ts

```diff
@@ -0,0 +1,56 @@
+import crypto from "node:crypto";
+import type { APIInteraction } from "discord-api-types/v10";
+import type { NextFunction, Request, Response } from "express";
+import { verify, PlatformAlgorithm } from "../node.js";
+import type { SubtleCryptoImportKeyAlgorithm } from "../types/index.js";
+
+/**
+ * Returns middleware for Express.js that validates a request from Discord.
+ * If you are not on the latest version 16 or 18 of Node, you should pass a
+ * specific value in for the algorithm.
+ *
+ * The returned middleware should be called before any other middleware, and
+ * functionally works the same as a body parser middleware such as
+ * `express.json`.
+ *
+ * @param publicKey - The application's public key
+ * @param algorithm - The name of the crypto algorithm to use
+ * @returns An express.js middleware function that verifies a request from Discord.
+ */
+export default function getMiddleware(
```

Comment:
What do you think about `verifyInteractionMiddleware` instead? I'm not super sure what the existing patterns are, so that might be "wrong"

Reply:
Sounds like a good idea, though after some looking around on NPM `verifyInteraction` seems like it'd be more in line with the ecosystem in general so we'll go with that.